src/main/java/net/dv8tion/jda/core/entities/Member.java

```diff
@@ -24,16 +24,52 @@
 import java.time.OffsetDateTime;
 import java.util.List;
 
+/**
+ * Represents a Guild-specific User.
+ * <p>
+ * Used to get data about this user's presence within a {@link net.dv8tion.jda.core.entities.Guild Guild}.
```

Comment:
This is actually more than just the User's presence. It contains "all guild-specific information about a User"